clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -117,49 +121,54 @@ with open(INVITE_CSV, "w") as f:
 null = open("/dev/null", "w+")
 
 common_args = ["-key", "home_domain", args.domain,
-               "-t", "t1",
-               args.target]
+        "-t", "t1"]
 
 initial_reg_cmd = [SIPP_BIN,
-                   "-sf", SCRIPTS_DIR + "register.xml",
-                   "-inf", REGISTER_CSV,
-                   "-trace_err", "-error_file", INITIAL_REG_ERRORS,
-                   "-r", str(args.initial_reg_rate),
-                   "-m", str(args.subscriber_count)] + common_args
+        "-sf", SCRIPTS_DIR + "register.xml",
+        "-inf", REGISTER_CSV,
+        "-trace_err", "-error_file", INITIAL_REG_ERRORS,
+        "-default_behaviors", "all,-bye",
+        "-r", str(args.initial_reg_rate),
+        "-m", str(args.subscriber_count)] + common_args + [args.icscf_target]
```

Comment:
These realignments may be deliberate?! I don't like it, I don't know why it's happened.